src/ansible_compat/runtime.py

```diff
@@ -0,0 +1,89 @@
+"""Ansible runtime environment maanger."""
```

Comment:
s/aa/a/

Reply:
is not a mistake is on purpose! See https://en.wikipedia.org/wiki/Manger ;)

Reply:
But the wiki page shows a word with just one "a" :)